src/storage/createAtId.js

```diff
@@ -1,40 +0,0 @@
-const middy = require("middy");
-const { cors } = require("middy/middlewares");
-const { uploadDocumentAtId } = require("./documentService");
-const { CORS_POLICY_HEADER } = require("../../utils/cors");
-
-const handleCreateAtId = async (event) => {
-  try {
-    const { document, ttl } = JSON.parse(event.body);
-    const { id } = event.pathParameters;
-    const receipt = await uploadDocumentAtId(document, id, ttl);
-    return {
-      statusCode: 200,
-      body: JSON.stringify(receipt),
-    };
-  } catch (e) {
```

Comment:
this went missing?

Reply:
the catch? automatically caught by the unknown error handler